### PR TITLE
Ydbdocs 828 topics transactions

### PR DESCRIPTION
**Pull request by ElenaAfina**
### Changelog entry 

Adds documentation about transactions with topics (RU)

### Changelog category 

* Documentation

### Additional information

Adds description about transactions with topics: concept, main principles and constraints.

### CODE REVIEW

File: ydb/docs/ru/core/concepts/topic.md [hunk 1]
```diff
@@ -151,3 +189,18 @@
...
+
+{{ ydb-short-name }} поддерживает работу с топиками в рамках транзакции. Все транзакции с топиками имеют уровень *Serialazable*
+
+### Транзакционное чтение из топика {#topic-transactions-read}
+
+Данные в топиках не меняются при чтении из топика, поэтому при транзакционном чтении из топика необходимо только транзакционно поменять офсет. SDK скрывает от пользователя детали, при транзакционном чтении через SDK одновременно происходит чтение и коммит офсета.
+
+### Транзакционная запись в топик {#topic-transactions-write}
+
+При транзакционной записи в топик данные промежуточно сохраняются в служебной партиции и затем публикуются (становятся видимыми) при коммите транзакции.
```
Comment (Alek5andr-Kotov) [inline ydb/docs/ru/core/concepts/topic.md:202]: Служебная партиция это деталь реализации. Её нужно показывать?

File: ydb/docs/ru/core/concepts/topic.md [hunk 2]
```diff
@@ -151,3 +189,18 @@
 
 Также ограниченно поддержан протокол Kafka API версии 3.4.0. (см. [Работа с Kafka API](../reference/kafka-api/index.md)).
 
+## Транзакции с учаcтием топиков {#topic-transactions}
+
+{{ ydb-short-name }} поддерживает работу с топиками в рамках транзакции. Все транзакции с топиками имеют уровень *Serialazable*
+
+### Транзакционное чтение из топика {#topic-transactions-read}
+
+Данные в топиках не меняются при чтении из топика, поэтому при транзакционном чтении из топика необходимо только транзакционно поменять офсет. SDK скрывает от пользователя детали, при транзакционном чтении через SDK одновременно происходит чтение и коммит офсета.
```
Comment (Alek5andr-Kotov) [inline ydb/docs/ru/core/concepts/topic.md:198]: Во втором предложении части не согласуюся друг с другом.

**Comment by blinkov** [comment]
@ElenaAfina, this looks like a duplicate of #9243 to a large extent; please separate the changes related to autopartitioning and transactions.

**Comment by ElenaAfina** [comment]
This PR not for publishing, can be deleted. Was created by error not from main.

**Pull request closed without merging**